devito/passes/clusters/aliases.py

```diff
@@ -3,10 +3,66 @@
 from sympy import Indexed
 import numpy as np
 
-from devito.ir.support import IterationInstance, LabeledVector, Stencil
-from devito.symbolics import retrieve_indexed
+from devito.ir import (ROUNDABLE, DataSpace, IterationInstance, IterationSpace,
+                       Interval, IntervalGroup, LabeledVector, Stencil,
+                       detect_accesses, build_intervals)
+from devito.logger import perf_adv
+from devito.passes.clusters.utils import dse_pass, make_is_time_invariant
+from devito.symbolics import estimate_cost, retrieve_indexed
+from devito.types import Array, Eq
+
+__all__ = ['cire']
+
+
+MIN_COST_ALIAS = 10
```

Comment:
Still can't be user modified easily right?

Reply:
no, but we can easily turn it into a dict, or better, provide an API. For another PR, I'd say